src/locales/en/translations.json

```
@@ -103,13 +103,15 @@
   "career.leadbox.subtitle": "Unsolicited Application",
   "career.leadbox.text": "Georgios Kaleadis, CTO",
   "career.leadbox.mail": "career@satellytes.com",
-  "office.title": "Office",
-  "office.heading": "Our new Office directly next to Sendlinger Tor",
-  "office.team.title": "Team",
-  "office.team.heading": "That is who we are",
-  "office.team.text": "Our experienced team of engineers and designers",
-  "office.leadbox.title": "Did you get interested?",
-  "office.leadbox.link": "Career",
+  "about-us.title": "About Us",
+  "about-us.office.title": "Office",
+  "about-us.office.heading": "Our new Office directly next to Sendlinger Tor",
+  "about-us.office.text": "Feel at home with us. A small impression of our office in Sendlinger Straße in the heart of Munich.",
+  "about-us.team.title": "Team",
+  "about-us.team.heading": "That is who we are",
+  "about-us.team.text": "Our experienced team of engineers and designers",
+  "about-us.leadbox.title": "Did you get interested?",
```

I know this is not necessarily related to this PR but these translations seem fairly "holprig". I suggest "This is who we are" instead of "That is who we are" as well as "Are you interested?" instead of "Did you get interested?":
```suggestion
  "about-us.team.heading": "This is who we are",
  "about-us.team.text": "Our experienced team of engineers and designers",
  "about-us.leadbox.title": "Are you interested?",
```

Some background:
- "That" implies distance. "This is my body, that is your body". "This" is close, "that" is distant. In this case the images of the team are right below the text so there is no distance thus we should be using "this".
- "Did you get interested?" is Denglisch. "Are you interested?" is (in my opinion) more proper translation.

---

thanks for the catches. right now DE is the primary language and we plan to send the texts to a translator once the german version is approved/ready. But I still would like to fix things like this so I very appreciate your suggestion 👍 The translations could be weird because of deepl and myself looking over every translation but only to catche absolute non-sense. I guess this slipped.